rllib/utils/framework.py

```diff
@@ -9,6 +9,17 @@
 TensorType = Any
 
 
+def get_auto_framework():
+    """Returns a framework (str) to use automatically.
+
+    If only PyTorch is installed, returns "torch", otherwise "tf".
```

Comment:
Can we also INFO log on auto detection? This could be very confusing otherwise.

Reply:
+1

Reply:
done